### PR TITLE
new account page

### PR DESCRIPTION
**Pull request by UnsungHero97**
Depends on:
https://github.com/openlattice/lattice-js/pull/1069
https://github.com/openlattice/lattice-sagas/pull/673

### CODE REVIEW

File: src/containers/account/AccountContainer.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  const thisUserInfo :UserInfo = AuthUtils.getUserInfo() || {};
+
+  useEffect(() => {
+    dispatch(getAtlasCredentials());
+  }, [dispatch]);
+
+  useEffect(() => () => {
+    dispatch(clearAtlasCredentials());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```
Comment (solomontang) [inline src/containers/account/AccountContainer.js:54]: include dispatch as a dep

File: src/containers/account/AccountContainer.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+          <Typography component="h2" variant="body2">USER ID</Typography>
+          <ElementWithButtonGrid fitContent>
+            <Pre>{thisUserInfo.id}</Pre>
+            <CopyButton onClick={() => clipboardWriteText(thisUserInfo.id)} />
+          </ElementWithButtonGrid>
+        </StackGrid>
+        <StackGrid gap={4}>
+          <Typography component="h2" variant="body2">AUTH0 JWT TOKEN</Typography>
+          <ElementWithButtonGrid fitContent>
+            {PasswordInput}
```
Comment (solomontang) [inline src/containers/account/AccountContainer.js:90]: What's the reason for not using the actual value in `<input type="password" />`

**Review by solomontang — APPROVED** [review]

File: src/containers/account/AccountContainer.js (hunk 1 above)
Comment (UnsungHero97) [reply src/containers/account/AccountContainer.js:54]: Will do.

File: src/containers/account/AccountContainer.js (hunk 2 above)
Comment (UnsungHero97) [reply src/containers/account/AccountContainer.js:90]: No particular reason. The input is there just for visuals and serves no other purpose, so I chose not to give it a value.

File: src/containers/account/AccountContainer.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        </div>
+        <div>
+          <Typography component="h2" variant="body2">EMAIL</Typography>
+          <Typography>{thisUserInfo.email || DASHES}</Typography>
+        </div>
+        <StackGrid gap={4}>
+          <Typography component="h2" variant="body2">USER ID</Typography>
+          <ElementWithButtonGrid fitContent>
+            <Pre>{thisUserInfo.id}</Pre>
+            <CopyButton onClick={() => clipboardWriteText(thisUserInfo.id)} />
```
Comment (bergieboy) [inline src/containers/account/AccountContainer.js:83]: looks like this is an icon button? `aria-label` plz.

File: src/containers/account/AccountContainer.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+          </ElementWithButtonGrid>
+        </StackGrid>
+        <StackGrid gap={4}>
+          <Typography component="h2" variant="body2">AUTH0 JWT TOKEN</Typography>
+          <ElementWithButtonGrid fitContent>
+            {PasswordInput}
+            <CopyButton onClick={() => clipboardWriteText(AuthUtils.getAuthToken())} />
```
Comment (bergieboy) [inline src/containers/account/AccountContainer.js:90]: looks like this is an icon button? `aria-label` plz.

File: src/containers/account/AccountContainer.js [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+          </ElementWithButtonGrid>
+        </StackGrid>
+        {
+          getAtlasCredentialsRS === RequestStates.SUCCESS && (
+            <>
+              <StackGrid gap={4}>
+                <Typography component="h2" variant="body2">ATLAS USERNAME</Typography>
+                <ElementWithButtonGrid fitContent>
+                  {PasswordInput}
+                  <CopyButton onClick={() => clipboardWriteText(atlasCredentials.get('username'))} />
```
Comment (bergieboy) [inline src/containers/account/AccountContainer.js:100]: looks like this is an icon button? `aria-label` plz.

File: src/containers/account/AccountContainer.js [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+                </ElementWithButtonGrid>
+              </StackGrid>
+              <StackGrid gap={4}>
+                <Typography component="h2" variant="body2">ATLAS CREDENTIAL</Typography>
+                <ElementWithButtonGrid fitContent>
+                  {PasswordInput}
+                  <CopyButton onClick={() => clipboardWriteText(atlasCredentials.get('credential'))} />
```
Comment (bergieboy) [inline src/containers/account/AccountContainer.js:107]: looks like this is an icon button? `aria-label` plz.

**Review by bergieboy — CHANGES_REQUESTED** [review]

**Review by giuliacampana — DISMISSED** [review]

**Review by giuliacampana — APPROVED** [review]

**Review by bergieboy — APPROVED** [review]

**Pull request merged**